docs/Docs_Manual_QA.md

```diff
@@ -13,3 +13,7 @@ XXX Test Passed: <link to the test results in your GitHub repository>
 </br>
 XXX Test Failed: <link to the test results in your GitHub repository>, <link to the issue>
 </br>
+
+Create a Project, Test passed: https://github.com/hridaya423/HacktoberfestMindsDBTesting/blob/master/Testing.md
+Remove a Project, Test passed: https://github.com/hridaya423/HacktoberfestMindsDBTesting/blob/master/Testing.md
+List Projects, Test Failed: https://github.com/hridaya423/HacktoberfestMindsDBTesting/blob/master/Testing.md, https://github.com/mindsdb/mindsdb/issues/7519
```

Comment:
Please update the file by removing test failed and the github issue and add that the test passed

Reply:
How do i edit it?